00XX-random-CG-election.md

```diff
@@ -0,0 +1,91 @@
+
+This HIP proposes the following changes to the CG election process:
+- New “CG score” attribute will be created for hotspots
+- A value between 0.0 and 1.0 will be randomly assigned for CG score once per CG election to each hotspot
+- “CG score” will replace the use of the current score in the CG election for filtering and weighting hotspots
+
+No other changes to how score and geography is used in elections are to be made as part of this HIP. 
+
+Due to the fact that CG score will be randomly assigned, there is no need to expose at his point to users of the Helium app or Explorer.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+Randomized selection of CG members may lead to unexpected consequences and impact performance of the CG.
+- Randomized selection may result in more CG members who perform poorly. Hotspots with high scores  are the ones most likely to be elected to CG. And hotspots with high score likely have those high scores because the owner has taken deliberate steps to optimize their setup (or even game the network). It is reasonable to assume that deliberate attention to the hotspot setup results in better performance of CG members. If too many poor performing CG members are randomly elected under this proposal, it make stall the blockchain or otherwise hurt performance.
+- This proposal provides more incentive to run hotspots without providing meaningful coverage. For example, "lone wolf" hotspots (a hotspot with no others in range) will, on average, see higher rewards. This may have undesirable effects on the growth of the network.
+
+# Rationale and Alternatives
+[alternatives]: #rationale-and-alternatives
+
+There are several alternatives to solving the core issue identified in this HIP: the disconnect between current hotspot score and CG performance. This HIP attempts to balance simplicity of implementation with fairness and providing for a path to a more comprehensive solution. Several alternatives are outlined below.
+
+By not taking up this proposal, CG rewards will continue to add incentive for hotspots to game PoC. Falsely asserting location and other gaming of PoC hurts the quality of the network and hurts honest hotspot owners. While this does not eliminate this gaming, it makes it slightly less profitable and starts to pave the way for a more robust CG approach.
+
+Additionally, it should be noted that PoC and CG have distinct portions of the HNT reward pool. As such, there should be a distinct mechanism for rewarding these. However the current scoring mechanism, if functioning properly, tightly couples these rewards: better PoC performance => higher score =>  greater chance to CG membership => more CG reward.
+
+### Alternative: Randomly assign hotspot score
```

Comment:
This alternative and the below alternative and the actual proposal are essentially the same, I think that these can be removed.